src/runtime/lpython_builtin.py

```diff
@@ -1085,6 +1085,64 @@ def _lpython_str_isspace(s: str) -> bool:
             return False
     return True
 
+@overload
+def _lpython_str_center(s: str, width: i32, fillchar: str) -> str:
+    """
+    Return centered in a string of length width. 
+    Padding is done using the specified fillchar (default is an ASCII space). 
+    The original string is returned if width is less than or equal to len(s).
+    """
+    if(len(fillchar) != 1):
+        raise TypeError("The fill character must be exactly one character long")
+    if(width <= len(s)):
+        return s
+    width -= len(s)
+    result: str = ""
+    left_padding: i32 = i32(width/2) + _mod(width,2)
+    i: i32 
+    for i in range(left_padding):
+        result += fillchar
+    right_padding: i32 = width - left_padding
+    result += s
+    for i in range(right_padding):
+        result += fillchar
+    return result
+
+@overload
+def _lpython_str_center(s: str, width: i32) -> str:
+    return _lpython_str_center(s, width, ' ')
+
+@overload
+def _lpython_str_expandtabs(s: str, tabsize: i32) -> str:
+    """
+    Return a copy of the string where all tab characters are replaced 
+    by one or more spaces, depending on the current column and the given tab size.
+    """
+    if len(s) == 0:
+        return s
+    col: i32 = 0
+    result: str = ""
+    c: str
+    for c in s:
+        if c == '\t':
+            if tabsize > 0:
+                i: i32
+                iterations: i32 = tabsize - _mod(col,tabsize)
+                for i in range(iterations):
+                    result += ' '
```

Comment:
For now this is fine, but we can later refactor this to simply use `*` operator.